share/completions/apk.fish

```diff
@@ -6,8 +6,8 @@ complete -f -c apk -n "__fish_seen_subcommand_from del fix version manifest" -a
 
 # Global options
 complete -f -c apk -s h -l help -d "Show help"
-complete -x -c apk -s p -l root -d "Install packages to DIR"
-complete -x -c apk -s X -l repository -d "Use packages from REPO"
+complete -F -c apk -s p -l root -r -d "Install packages to DIR"
```

Comment:
We usually only use `-F` if really necessary. I think in this case you can just drop it (untestd), since file completion is the default.

Reply:
Didn't test all but it seems to work fine